buaathesis.cls

```diff
@@ -275,7 +284,7 @@ The LaTeX template for thesis of BUAA]
     bookmarksopen,
     pdftitle={BUAA thesis},
     pdfauthor={BHOSC},
-    pdfsubject={北航毕业设计论文},
+    pdfsubject={北航毕业设计开题报告/文献综述},
     pdfcreator={LaTeXed~By~BHOSC}
 }
```

Comment:
Using a if-else block here

```
\ifbuaa@ktreport
    \hypersetup{
        bookmarksnumbered,
        bookmarksopen,
        pdftitle={BUAA thesis},
        pdfauthor={BHOSC},
        pdfsubject={北航毕业设计开题报告/文献综述},
        pdfcreator={LaTeXed~By~BHOSC}
    }
\else
    \hypersetup{
        bookmarksnumbered,
        bookmarksopen,
        pdftitle={BUAA thesis},
        pdfauthor={BHOSC},
        pdfsubject={北航毕业设计论文},
        pdfcreator={LaTeXed~By~BHOSC}
    }
\fi
```

would be better

Reply:
If statement added already.